apps/spruce/src/hooks/useScrollToAnchor/index.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useRef } from "react";
+
+/**
+ * `useScrollToAnchor` scrolls to an anchor element on the page if the URL contains an anchor.
+ */
+const useScrollToAnchor = () => {
```

Comment:
I think it would be good to have tests for this hook. Maybe the hook can keep track of a `hasScrolled` which can make it easier to test.